lib/InputWithButton/index.js

```diff
@@ -1,8 +1,32 @@
 import React, { Component } from 'react';
-import cx from 'classnames';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import Input from '../core/Input';
 import Button from '../Button';
 
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const LeftHandInput = styled(Input)`
+  width: 220px;
+  border-radius: 4px 0px 0px 4px;
```

Comment:
So where should we define `consts`? 😏 

Ideally, we should mostly map what we have now I think.

Reply:
Nvm, just saw the `theme` stuff 👍

Reply:
I realised you wrote it in the PR but I kind of wanted a super fresh opinion without any pre-knowledge, thanks for writing the description etc.